pkgs/applications/blockchains/miniscript/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "miniscript";
+  version = "unstable-20201104";
+
+  src = fetchFromGitHub {
+    owner = "sipa";
+    repo = pname;
+    rev = "5115dd0bdf70c3e3b728c2c0e9c8af4b7edd13ed";
+    sha256 = "0bklhlnl4wkb90rpqhs5n0h6rmnyb3js8ldgynppn27b2kbbmik5";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp miniscript $out/bin/miniscript
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Miniscript is a language for writing (a subset of) Bitcoin Scripts in a structured way, enabling analysis, composition, generic signing and more.";
```

Comment:
This is description is quite long. It should probably be moved to ``longDescription`` and a shorter one for the short should be used.